bcgov/HOUS-permit-portal
# Feature/bphh 220 pids and fields

## Description

Added pids / pins / full_address onto the application.  This is to support pulling from geocoder later on.  The UI is not complete for htis, waiting on mocks from submitter flow, but this will allow us to be unblocked on working on other integration pieces.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-220

## Steps to QA

Run seeds if there are no permit applications.  You should be able to find 4 permit applications for North vancouver.  They should have valid full addresses and pids.  You should be able to see this for the submitter.

## UI Accessibility Checklist

Not relevant here.

## General Checklist
